docker-compose.yml

```
@@ -72,6 +72,7 @@ services:
       - ./web/nginx:/etc/nginx/templates
       - ./web/alloy:/etc/nginx/alloy
       - ./web/certbot/conf:/etc/letsencrypt
+      - /var/log/nginx:/var/log/nginx
```

```suggestion
      - ./web/nginx/log:/var/log/nginx
```
lets use this relative directory

---

change to the directory above. will need to check if the directory is created. dont think it will because the master branch on zap-me repo doesn't say to change to this directory yet.

---

once this is merged on the master zap-me it should start creating the dir above and the logs as well.

---

corrected to match the above.